README.md

```diff
@@ -274,3 +274,63 @@ failed:
 ```
 
 </details>
+
+## Logic preview
+<details>
+<summary>
+Logic diagrams
+</summary>
+
+```mermaid
+graph TD
+    A[WORKFLOW DOKKU] -->|Deploy to Dokku| B(activate tarantool instance 'job rerunner')
+    B --> C[Runs function 'webhook_handler', a function that decides <br />whether to restart jobs or not]
+    C -->|perform check 1| D{needs_restart?}
+    C -->|perform check 2| E{is_job_failed?}
```

Comment:
We don't have these functions anymore. Let's remove all diagrams for now and then add them in the next PR.

Reply:
Done